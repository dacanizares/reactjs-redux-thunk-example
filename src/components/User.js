import React from 'react'
import { Card, CardImg, CardBody, CardText } from 'reactstrap'

const User = ({avatarUrl, name, email}) => {
  return(
    <Card>
      <CardImg top src={avatarUrl} alt={name} />
      <CardBody>
        <h3>{name}</h3>
        <CardText>
            {email}
        </CardText>
      </CardBody>
    </Card>
  );
}

export default User;