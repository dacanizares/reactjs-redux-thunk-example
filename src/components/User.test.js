import React from 'react'
import renderer from 'react-test-renderer'
import User from './User'

it('renders correctly', () => {
  const tree = renderer
    .create(<User name='Title' email='my@email.com' avatarUrl='ImageUrl' />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
