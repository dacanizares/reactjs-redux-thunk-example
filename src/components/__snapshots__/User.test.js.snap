// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="card"
>
  <img
    alt="Title"
    className="card-img-top"
    src="ImageUrl"
  />
  <div
    className="card-body"
  >
    <h3>
      Title
    </h3>
    <p
      className="card-text"
    >
      my@email.com
    </p>
  </div>
</div>
`;
